services/concepts/failover-protection.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Learn about failover protection
+sidebar_position: 1
+---
+
+# Failover protection
+
+:::info
+
+Failover support is available on Mainnet only.
+
+:::
+
+Failover protection is available on the Mainnet network for customers on the Growth or Custom plans.
+This feature ensures API requests are fulfilled even if Infura experiences a service issue or
+outage. This is achieved by routing requests to a select partner.
+
+Failover protection is enabled on a per-request basis by adding the failover header to your API
+request (see [Enable API request forwarding](../how-to/enable-api-forwarding.md)). This gives
+you control over which requests you'd like fulfilled by Infura's partner if failover protection should be needed.
+
+## Partners and privacy policies
```

Comment:
As a general topic this needs updating. For example, we'll need to list which networks supports Failover.
Also, different partners support different failover networks.

Reply:
Thank you @bgravenorst. There are several sections where the direct transfer of information from the Infura docs to MetaMask requires some refinement (or simply doesn't fit well). I started a Google doc when I was doing the restructure, to identify content that may need some "MetaMasking". Maybe we can review it as a team and potentially create some tickets to tackle issues in the future.  For instance, somethings I mention are related to: 

- How we present Infura within MetaMask (RPC nodes for MM)
- Which tutorials are relevant to a user (I'm not sure, but some of the Infura tutorials look like they don't have much to do with MM)
- Where and when to add  testnet documentation (it's presently omitted in MM etc. 

I think we will just iteratively work on it, once the initial doc set is merged.